src/core/geometry/qgsrectangle.h

```diff
@@ -295,6 +295,7 @@ class CORE_EXPORT QgsRectangle
      */
     void grow( double delta )
     {
+      if ( isNull() ) return;
```

Comment:
```suggestion
      if ( isNull() )
        return;
```

Reply:
shouldn't the prepare-commit hook be dealing with these already ?
I'd be happy to commit your suggested changes but if I do I'm afraid CI will need to run again and thus merging this would be posticipated by another day, worth it ?

Reply:
> shouldn't the prepare-commit hook be dealing with these already ?

PRs welcome.

Reply:
> PRs welcome.

I'll check if astyle is not already being run, but I'm pretty sure I often see it correcting my style so it must be, does it mean it's not dealing with naming conventions ?

Reply:
That's correct

Reply:
Ok I've applied your suggestion and pushed as a separate commit. I didn't rebase and squash to avoid loosing your review